php/example.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+require 'vendor/autoload.php';
+
+use Ramsey\Uuid\Uuid;
+
+$client = new GuzzleHttp\Client();
+
+# Credentials
+$apikey = $_SERVER["APIKEY"];
+$secret = $_SERVER["SECRET"];
+
+echo $apikey;
+# Get current offers
+
+if (!$apikey || !$secret) {
+//  throw new Error('apikey and/or secret is required');
+}
+
+echo "Current best offers:\n";
+$res = $client->request('GET', 'https://api.walutomat.pl/api/v1/public/market/orderbook/EUR_PLN');
+echo $res->getBody();
+
+
+# Check my balance
+$uri = '/api/v1/account/balances';
+$ts = round(microtime(true) * 1000);
+$sign = hash_hmac("sha256", $uri.$ts, $secret);
+$headers = [
+  'X-API-KEY' => $apikey,
+  'X-API-NONCE' => $ts,
+  'X-API-SIGNATURE' => $sign
+];
```

Comment:
w produkcyjnej wersji jednak lepiej mieć generowanie headersów w jednym miejscu. na potrzeby przykładu pewnie styknie ale znając życie korzystający z przykładu będą to kopiować

Reply:
bycmoze lepiej byloby przerobic wszystkie przyklady zeby zawierały prostą funkcję podpisującą headery